cognite/client/data_classes/capabilities.py

```diff
@@ -797,6 +797,22 @@ class Scope:
         DataSet = DataSetScope
 
 
+@dataclass
+class LocationFiltersAcl(Capability):
+    _capability_name = "locationFiltersAcl"
+    actions: Sequence[Action]
+    scope: AllScope | IDScope
+    allow_unknown: bool = field(default=False, compare=False, repr=False)
+
+    class Action(Capability.Action):  # type: ignore [misc]
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+        SpaceID = IDScope
```

Comment:
Have you tested this scope? IDScope is named `idScope` and looking at the spec it seems `id` is the correct name.
![image](https://github.com/user-attachments/assets/f8c58066-6314-4e86-8787-f8f37d42b08a)

Reply:
I think the "spec" is inaccurate, this is the json sent by Fusion: 

```{
            "locationFiltersAcl": {
              "actions": [
                "READ",
                "WRITE"
              ],
              "scope": {
                "idScope": {
                  "ids": [
                    "1"
                  ]
                }
              }
            }
          }```